packages/cli/src/lib/check-lockfile.test.ts

```diff
@@ -1,39 +1,30 @@
 import {checkLockfileStatus} from './check-lockfile.js';
 import {describe, it, expect, vi, beforeEach, afterEach} from 'vitest';
-import {file, path, git, outputMocker} from '@shopify/cli-kit';
-
-vi.mock('@shopify/cli-kit', async () => {
-  const cliKit: any = await vi.importActual('@shopify/cli-kit');
-
-  return {
-    ...cliKit,
-    git: {
-      factory: vi.fn(),
-    },
-  };
-});
+import {writeFile, inTemporaryDirectory} from '@shopify/cli-kit/node/fs';
+import {joinPath} from '@shopify/cli-kit/node/path';
+import {checkIfIgnoredInGitRepository} from '@shopify/cli-kit/node/git';
+import {mockAndCaptureOutput} from '@shopify/cli-kit/node/testing/output';
 
 describe('checkLockfileStatus()', () => {
   const checkIgnoreMock = vi.fn();
-  const gitFactoryMock = {
-    checkIgnore: checkIgnoreMock,
-  };
 
   beforeEach(() => {
-    vi.mocked(git.factory).mockReturnValue(gitFactoryMock as any);
+    vi.mock('@shopify/cli-kit/node/git');
+    vi.mocked(checkIfIgnoredInGitRepository).mockImplementation(
+      checkIgnoreMock,
+    );
     vi.mocked(checkIgnoreMock).mockResolvedValue([]);
   });
 
   afterEach(() => {
     vi.restoreAllMocks();
-    outputMocker.mockAndCaptureOutput().clear();
```

Comment:
I think we want to keep this but change to `mockAndCaptureOutput().clear()`?

Reply:
Good catch! It looks like `mockAndCaptureOutput` has side effects (it writes to a global variable), so better to call clear 👍